### DISCUSSION TITLE
How to override Component Level description text in DocsPage?

### DISCUSSION THREAD

**Discussion by peterlau** [storybookjs/storybook#16113] [Help] [answered]
I see that DocsPage extracts the description text from the JSDoc comment block associated with the component implementation e.g.
```
/** 
* This is a my description text.
*/
export const Button = (props) => ...
```

However I can't figure out how to override this text **without** abandoning the default DocsPage and writing MDX manually.
In particular, I've tried this technique from the `addon-info` documentation but it appears to not be observed with `addon-docs`.

In `Button.stories.jsx`:
```
import { Button } from "./Button";
export default {
  title: 'Components/Button',
  component: Button,
  parameters: {
    info: {
      text: "My overridden description" // Does not work.
    }
  }, 
...
}
```

So the question is - is there a way to override the description **without** modifying the comment in the component itself, and  **without** resorting to manually the doc page with MDX?

Thanks

**Comment by shilman** [comment, marked as answer]
```js
export default { 
  title: 'Components/Button',
  component: Button,
  parameters: {
    docs: {
      description: {
        component: 'My overridden description',
      }
    }
  }
}
```

cc @jonniebigodes

**Reply by jonniebigodes** [reply]
@peterlau indeed you can and as @shilman pointed out that's how to do it. We have it documented [here](https://storybook.js.org/docs/react/writing-docs/doc-blocks#docspage-2).

**Reply by peterlau** [reply]
Thanks all for pointing this out. I appreciate the help.

**Reply by jonniebigodes** [reply]
No need to thank whatsoever. Glad that we're able to help out! 

Feel free to reach out to us with any further questions here or on our Discord Server.

Stay safe
